src/test/java/PasswordGeneratorTest.java

```diff
@@ -49,10 +48,12 @@ public void checkPasswordLength() {
 
     @Test
     public void checkSecretPassword() {
+        //do we need this variable?
         BCryptPasswordEncoder bCryptPasswordEncoder = new BCryptPasswordEncoder();
```

Comment:
We already have a BCryptPasswordEncoder variable, we shouldn't need to create another one.